python/GafferCyclesTest/IECoreCyclesPreviewTest/RendererTest.py

```diff
@@ -1429,5 +1433,112 @@ def testSurfaceAttributeWithGenericShaderType( self ) :
 
 		del plane
 
+	def __valueAtUV( self, image, uv, channelName = "" ) :
+
+		dimensions = image.dataWindow.size() + imath.V2i( 1 )
+
+		ix = int( uv.x * ( dimensions.x - 1 ) )
+		iy = int( uv.y * ( dimensions.y - 1 ) )
+		i = iy * dimensions.x + ix
+
+		return image[channelName][i]
+
+	def testCustomAOV( self ) :
+
+		renderer = GafferScene.Private.IECoreScenePreview.Renderer.create(
+			"Cycles",
+			GafferScene.Private.IECoreScenePreview.Renderer.RenderType.Interactive,
+		)
+
+		# Custom AOVs are currently not supported in OSL mode.
+		# See https://developer.blender.org/T73266 for further updates
+		# for when OSL will eventually support custom AOVs.
+		renderer.option( "cycles:shadingsystem", IECore.StringData( "SVM" ) )
+
+		renderer.output(
+			"testValueOutput",
+			IECoreScene.Output(
+				"testValue",
+				"ieDisplay",
+				"aovv myValueAOV",
```

Comment:
Is there a specific reason for using the `aovv` and `aovc` syntax rather than the `float` and `color` we use for outputs in other renderers?

Reply:
Sounds good to me! I was wondering how I'd support a `vector` custom aov type, according to one of the roadmap points that Cycles has with the v already taken.